layouts/_default/article.html

```diff
@@ -1,49 +1,49 @@
-<div class="article ">
-    <a class="block" href="{{ .Permalink }}">
-        <img loading="lazy" src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl object-cover" style="height:212px; width:100%"></a>
-    <div class="bg-[#ECF6FF] rounded-b-xl ">
-
-        <div class="px-6 pt-4">
-            <ul class="flex flex-wrap">
-                {{ range $key, $value := .Params.tags }}
-                    <li class="text-base font-bold text-[#1D65A6]">
-                        <a href="/tags/{{ $value | urlize }}">{{ $value }}</a>{{ if ne $key (sub ( len $.Params.tags) 1) }},&nbsp; {{ else }}{{ end }}
-                    </li> 
-                {{ end }}
-            </ul>
-            <h3 class="text-xl font-extrabold mt-3 md:mt-4 pb-2 "><a class="block h-auto lg:h-24" href="{{ .Permalink }}">{{ .Title }}</a>
+<div class="flex flex-col h-full">
+    <a href="{{ .Permalink }}">
+        <img loading="lazy" src="{{ .Params.image }}" alt="" class="rounded-t-xl object-cover" style="height:212px;width:100%"/>
+    </a>
+    <div class="px-4 pt-4">
+        <ul class="flex flex-wrap">
+          {{ range $key, $value := .Params.tags }}
+            <li class="text-base font-bold text-[#1D65A6]">
```

Comment:
If tags appear on more than 1 line this will cause the blog post into text to not appear on the same line. After attempting to adjust the height, it felt best to leave this style as is for now.

Reply:
Ok, that makes sense to me -- we can also try to shorten our tags if necessary or we can use the elipses thing that we do to truncate the text; I think more frequent use of tags (e.g. the same tag for multiple posts) is better for inter-post relationships rather than exhaustive tagging for SEO.